Frontend/src/router/index.js

```diff
@@ -21,13 +21,21 @@ const routes = [
   },
   {
     path: "/:year/fluege/",
-    name: "FlightsAll",
+    name: "FlightsAllYear",
     props: true,
     beforeEnter: validateRouteParamYear,
     meta: { toTop: true },
 
     component: () => import("../views/FlightsAll.vue"),
   },
+  {
+    path: "/fluege/",
+    name: "FlightsAll",
```

Comment:
Wollen wir wirklich eine Anzeige für alle Flüge, die jemals eingetragen wurden?

Reply:
Begründung siehe oben.

Ich will ja mit router-link arbeiten. Dann brauche ich den Eintrag hier. Generell würde ich aber immer noch andere Parameter mitgeben (z.B. Team-,Clubnamen, UserId, ...)

Verwendungsbeispiele sind in UserCard zu sehen.